confluence.go

```diff
@@ -88,6 +89,29 @@ func NewAPI(url, username, password string) (*API, error) {
 	}, nil
 }
 
+// NewAPIWithToken create new API struct using Token
+// See https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html
+// Support Confluence 7.9 and later
+func NewAPIWithToken(url, token string) (*API, error) {
+	switch {
```

Comment:
You have to add a check and error for an empty token value. And maybe an additional check for token length (it must be 44 symbols long).

Reply:
3946d9c c062f9b